skyportal/handlers/api/photometry.py

```diff
@@ -228,6 +228,7 @@ def serialize(
     annotations=True,
     owner=False,
     stream=False,
+    validation=False,
```

Comment:
What form of the word makes sense here: `validated` or `validate`?

Reply:
Oh semantics ;) validation was like the noun rather than the verb or past participle which is why I used it.